sdk/core/azure-core/azure/core/pipeline/transport/_aiohttp.py

```diff
@@ -215,6 +211,10 @@ def __len__(self):
         return self.content_length
 
     async def __anext__(self):
+        from requests.exceptions import (
```

Comment:
More generic question: why do we even import requests in this file in the first place? This is the aiohttp streamer, that sounds like a bug (copy/paste) more than anything: why would you get a `requests` exception from anywhere here in the first place?
I would take this opportunity to investigate.